docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml

```diff
@@ -1163,6 +1211,56 @@
               <entry>If false, disables attempted aggressive constant value
               optimizations.</entry>
             </row>
+
+            <row>
+              <entry><emphasis>exportDependencies</emphasis></entry>
+
+              <entry>Default: false</entry>
+
+              <entry>Generate information about inter-definition
+              dependencies</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>maxCompileThreads</emphasis></entry>
+
+              <entry>Default 2</entry>
+
+              <entry>Number of compiler instances to compile the c++</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>reportCppWarnings</emphasis></entry>
+
+              <entry>Default: false</entry>
+
+              <entry>Report warnings from c++ compilation</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>saveCppTempFiles</emphasis></entry>
+
+              <entry>Default: false</entry>
+
+              <entry>Retain the generated c++ files</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>spanMultipleCpp</emphasis></entry>
+
+              <entry>Default: false</entry>
+
+              <entry>Generate a work unit in multiple c++ files </entry>
+            </row>
+
+            <row>
+              <entry><emphasis>activitiesPerCpp</emphasis></entry>
+
+              <entry>Default 1</entry>
```

Comment:
Default is 500 for linux and 800 for windows.  I'm not sure how much detail is worth recording.